src/Sylius/Bundle/CoreBundle/Resources/config/app/workflow/sylius_product_review.yaml

```diff
@@ -0,0 +1,21 @@
+framework:
+    workflows:
+        sylius_product_review:
+            type: state_machine
+            marking_store:
+                type: method
+                property: status
+            supports:
+                - Sylius\Component\Core\Model\ProductReview
```

Comment:
Can we use here a parameter?

Reply:
We can't use a parameter because the Workflow doesn't see it. I added and used the `ProductReviewInterface`